examples/ext/cooldowns/basic.py

```diff
@@ -0,0 +1,27 @@
+import crescent
+from crescent.ext import cooldowns
+
+bot = crescent.Bot("...")
+
+
+# The user can use the command 3 times in 20 seconds.
+@bot.include
+@crescent.hook(cooldowns.cooldown(20, 3))
```

Comment:
I'm not sure if this is the fault of pycooldown, but I personally think the capacity should go before the period (so `cooldown(3, 20)` means 3 executions per 20 seconds`

Reply:
If this is because of pycooldown I think that's worth changing too - but I think you can just change the function. Just use kwargs when calling pycooldown to avoid having to bump above beta.7

Reply:
Yeah, I just copied the syntax for pycooldown.

Reply:
Actually I think period should stay first since its the more "thought about" aspect of a rate limit

Reply:
Im actually gonna switch it then, since i dont want something to feel like an annoying change from what people are used to